solutions/swb-app/src/projectEnvTypeConfigRoutes.ts

```diff
@@ -47,7 +48,14 @@ export function setUpProjectEnvTypeConfigRoutes(
           user: res.locals.user
         }
       );
-      await projectEnvTypeConfigService.associateProjectWithEnvTypeConfig(request);
+      try {
+        await projectEnvTypeConfigService.associateProjectWithEnvTypeConfig(request);
+      } catch (e) {
+        if (isProjectDeletedError(e)) {
+          throw Boom.badRequest(e.message);
+        }
+        throw e;
```

Comment:
We still want the route to throw a `Boom` error and just throwing `e` does not ensure that. Do this instead:
```
if (Boom.isBoom(e)) {
  throw e;
}
throw Boom.badImplementation(<helpfulMessage>);

```

Reply:
We should try to make sure any code changes use Boom :-D